actions/datadog-release-metric/DatadogReleaseMetric.ts

```diff
@@ -25,13 +27,23 @@ export default class DatadogReleaseMetric {
       throw new Error("This GitHub Action should only be run on tags");
     }
 
-    const params: MetricParams = {
-      type: METRIC_TYPE,
-      name: METRIC_NAME,
-      tags: this.getTags(this.getOrg(), this.getRepo(), this.isPreRelease()),
+    const tags = this.getTags(this.getOrg(), this.getRepo(), this.isPreRelease());
+
+    const release_count_params: MetricParams = {
+      type: METRIC_TYPE_COUNT,
+      name: METRIC_NAME_RELEASE_COUNT,
+      tags: tags,
       value: 1,
     };
-    await this.sendMetric(params);
+
+    const release_status_params: MetricParams = {
+      type: METRIC_TYPE_GAUGE,
+      name: METRIC_NAME_RELEASE_STATUS,
+      tags: tags,
+      value: 0, // This indicates the repo is no longer releasing. 
+    };
+
+    await Promise.all([this.sendMetric(release_count_params), this.sendMetric(release_status_params)]);
```

Comment:
nit: a single call to `datadog.submitMetrics()` accepts a series of values so _could_ refactor `sendMetric` to accept multiple metric params.

Reply:
I thought about that but didn't find much benefit of using either approach over the other. Since I have already made the edits to the tests accordingly, going forward with the implemented approach.